src/aatoolbox/datasources/usgs/usgs_ndvi.py

```diff
@@ -348,78 +313,32 @@ def process(
         # if clobber or check dates already processed
         # if not
         if processed_path.is_file():
-            df = self.load()
-
-            # check that the processed file has the same analyzed
-            # indicators and column for aggregation as passed
-            # to process()
-            cols = kwargs.get(
-                "stats_list", ["mean", "std", "min", "max", "sum", "count"]
+            dates_to_process, df = self._determine_process_dates(
```

Comment:
confusing that the output name of this is the same. Maybe change the output name of `_expand_dekads` to smth like `included_dates`?